go.mod

```diff
@@ -79,4 +80,6 @@ require (
 	sourcegraph.com/sourcegraph/appdash-data v0.0.0-20151005221446-73f23eafcf67
 )
 
+replace github.com/google/pprof => github.com/lonng/pprof v0.0.0-20191012154247-04dfd648ce8d
```

Comment:
oh, maybe it is a bad idea to use our own pprof fork? can we use internal pprof/directly?

Reply:
Take it easy, this fork just export some internal subpackage. The internal pprof uses the `github.com/google/pprof` directly.